### ISSUE TITLE
Startup folder always one level up in Emacs 24?

### ISSUE THREAD

**Issue by lionel-** [emacs-ess/ESS#468]

**Comment by pauljohn32** [comment]
I reported this in ess email list originally. Today I confirm the problem happens in Emacs-24.5 on Centos EL7 with a fresh install of ESS-17.11.

HOWEVER, the problem depends on the startup file.  This line causes the problem:

;; start R in current working directory, don't let R ask user:
(setq ess-ask-for-ess-directory nil)

Removing the init file, or commenting out just that one line, solves the problem.

The only Emacs in the 24 generation I have for testing is 24.5.

In current Ubuntu or Fedora, it is easy enough to upgrade to Emacs 25 and the problem disappears. However, I'm having quite a struggle building an RPM for Emacs 25 on the older EL7 systems here. I'll figure that out, I usually do :)

**Comment by lionel-** [comment]
@Currerius What version of Emacs are you using? Do you observe that behaviour only in git repos?

**Comment by Currerius** [comment]
After fresh install with Ubuntu 16.04.04, Emacs 24.5.1 and ESS 17.11-2 (version number from `apt show`) R started one folder up in the git-repo I was working on. I then tried another folder and R started in the current folder. I only understood the "project"/git issue after google searching this behavior. 

I am mostly working on my own without to much branching, such that I did not consider any kind of "project" paradigm, yet. The discussion around this issue made me curious though.

Thanks a ton for creating ESS - it is totally awesome! Especially polymode with Rmarkdown...
